README.md

```diff
@@ -1 +1,44 @@
-# mv-awesome-books
\ No newline at end of file
+# HTML-CAPSTONE-PROJECT
+
+> This project is a Microverse exercise on Javascript DOM manipulation, event listener, arrow function, let and const, Javascript Object, Arrays, String interpolation, local storage.
+
+
+![screenshot]()
```

Comment:
- [ ] Kindly fix the screenshot of the README file.
_Please notice that it's so important to have a professional README file for each of your projects._